doc.go

```diff
@@ -131,7 +136,7 @@ const indexTemplate = `
       		</select>
 		</label>
     </div>
-    <div class="col-6"></div>
+    <div class="col-6 custom-top-update">Last Updated At: {{.UpdatedAt}}</div>
```

Comment:
```suggestion
    <div class="col-6 custom-top-update">Last Updated: {{.UpdatedAt}}</div>
```

Reply:
I don't understand why such date is highlighted on top of the navigation bar. It's awkward IMHO. Is it important? Can we just put it at the bottom of the page? See https://golang.design/go-questions/errata/ as an example.

Reply:
yes, we should not highlighted on top of navigation bar, and put it at the bottom of the page.